### DISCUSSION TITLE
实用工具箱 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#104] [Announcements]
# 实用工具箱 - CS自学指南

CS自学指南

https://csdiy.wiki/%E5%BF%85%E5%AD%A6%E5%B7%A5%E5%85%B7/tools/

**Comment by polar9527** [comment]
https://learngitbranching.js.org
图形化快速上手git

**Comment by KinGeorge** [comment]
[菜鸟教程](https://www.runoob.com/)
简单的语言教程

**Reply by Flora-2017** [reply]
哈哈嗝，you are interesting.

**Comment by LittleRed007** [comment]
好奇折纸有什么用呢qwq

**Reply by PKUFlyingPig** [reply]
hhhh，可以送妹子～～

**Comment by SwingURM** [comment]
ecust-CourseShare: 华东师范大学研究生课程攻略共享计划

**Reply by SwingURM** [reply]
链接是对的 前面的学校名字不太对hh

**Comment by DentistCode** [comment]
Z-library：
> 2023年2月11日，用户可以再次访问Z-Library，不过用户要点击singlelogin.me链接并登录他们的帐户才能访问Z-Library ——维基百科
没想到这里竟然能直接访问，太棒了

**Comment by woqiqishi** [comment]
现在或许应该再加上ChatGPT

**Comment by guanjiesun** [comment]


**Reply by chencx2023** [reply]
卖片的都没码

**Comment by guanjiesun** [comment]
### 油管或者B站视频下载：[ParseVideo](https://www.parsevideo.com/)
### BitTorrent客户端：[Motrix](https://github.com/agalwood/Motrix)

**Reply by CS-Trekker** [reply]
滚

**Comment by Tingwk** [comment]
Library Genesis  is pretty nice

**Comment by shicxin** [comment]
这个页面单独收藏了😍

**Comment by Javien-Li** [comment]
https://annas-archive.org/   最大的在线图书馆.自带在线阅读网站,支持免费下载电子书,支持多种语言

**Comment by xielinhegel** [comment]
手把手教折纸这个是干啥用的

**Reply by CS-Trekker** [reply]
如其名

**Comment by Leokkkkkkk** [comment]
应该再加上cursor chatgpt and claude!

**Comment by xielinhegel** [comment]
各位大佬有没有mac键盘控制Windows的好方案

**Comment by xjn2005** [comment]
https://zh.annas-archive.gl/ 安娜档案馆，感觉这个比zlib更好用。

**Comment by KYO11978** [comment]
https://sci-hub.se/这个网站没法访问

**Comment by Zhaoan025** [comment]
这里提供的Z图书馆链接失效了现为https://zlib.li/，但需要挂梯子加速

**Reply by Zhaoan025** [reply]
第一次发，不太会QwQ，是https://zlib.li/
